state_management/javascript/sdk/order-processor/index.js

```diff
@@ -1,45 +1,41 @@
-import { DaprClient } from 'dapr-client'; 
+import { DaprClient } from 'dapr-client'
 
-const DAPR_HOST = process.env.DAPR_HOST || "http://localhost";
-const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT || "3500";
+const DAPR_HOST = process.env.DAPR_HOST ?? "http://localhost"
```

Comment:
split protocol from host, use protocol in constructing the url

Reply:
I split the protocol from host. See other comment for details.